_scalingo_api/events.md

```diff
@@ -1779,25 +1779,36 @@ _When:_ A maintenance is scheduled on your database
 `type=plan_database_maintenance`
 
 {:.table}
-| field                         | type      | description                     |
-| ------------------------------| --------- | ------------------------------- |
-| maintenance_id                | string    | The related maintenance ID      |
-| addon_name                    | string    | The related addon name          |
+| field                       | type     | description                                   |
+| --------------------------- | -------- | --------------------------------------------- |
+| maintenance_id              | string   | The related maintenance ID                    |
+| addon_name                  | string   | The related addon name                        |
+| maintenance_type            | string   | The type of maintenance executed              |
+| maintenance_window_in_hours | integer  | The maintenance window length (in hours)      |
+| next_maintenance_window     | datetime | The beginning of your next maintenance window |
 
 ||| col |||
 
 Example object:
 
 ```json
 {
-  "id":"62d13ad1a5b6ec0001e553c8",
-  "created_at":"2022-07-15T10:00:49.703Z",
-  "type":"plan_database_maintenance",
-  "app_id": "5343eccd646173000a140000",
-  "app_name": "appname",
+  "id": "650312574002c001afcdf988",
+  "created_at": "2023-09-14T14:01:59.916Z",
+  "type": "plan_database_maintenance",
+  "app_id": "649e9d0389bca600016ea61b",
+  "app_name": "sample-go-martini",
+  "user": {
+    "username": "scalingo-platform",
+    "email": "deploy@scalingo.com",
+    "id": "us-b32ca09d-1608-4e6f-8f14-302b447b0e14"
```

Comment:
**suggestion**: For consistency, I'd make the same as everywhere in this file, i.e. impersonate the user by using 
```
  "user": {
    "username": "johndoe",
    "email": "john@doe.com",
    "id": "us-0e6d8e46-5cd0-42a4-acba-372b2be605ac"
  },
```

Reply:
you're right, fixed